server/sonar-server/src/main/java/org/sonar/server/permission/PermissionPrivilegeChecker.java

```diff
@@ -42,9 +43,9 @@ public static void checkProjectAdminUserByComponentKey(UserSession userSession,
     }
   }
 
-  public static void checkProjectAdminUserByComponentUuid(UserSession userSession, @Nullable String componentUuid) {
+  public static void checkAdministrationPermission(UserSession userSession, Optional<ProjectId> projectId) {
```

Comment:
This method name makes me wonder what "administration permission" is.

Checking the 'admin' (value in DB) is referred to as "admin" pretty much everywhere and especially in the other methods of this class.

Why not simply call this method `checkProjectAdminUser` or `checkProjectAdmin`? (I prefer the later but less consistant with other methods of the class)